pkg/config/config.go

```diff
@@ -291,7 +294,7 @@ func (lc *LocalConfig) GetRef() string {
 }
 
 // GetSourceType returns the source type, returns default if nil
-func (lc *LocalConfig) GetSourceType() occlient.CreateType {
+func (lc *LocalConfig) GetSourceType() SrcType {
```

Comment:
I don't like how we're using SrcType... Why not SourceType?

Reply:
Its a name of field already in one of the structs in same file and so go complains if I name it  as SourceType in both places(here and a field in LocalConfigInfo)